sd/eureka/client.go

```diff
@@ -0,0 +1,83 @@
+package eureka
+
+import (
+	stdeureka "github.com/hudl/fargo"
+	stdeurekalogging "github.com/op/go-logging"
+)
+
+func init() {
+	// Quieten Fargo's own logging
+	stdeurekalogging.SetLevel(stdeurekalogging.ERROR, "fargo")
+}
```

Comment:
I know, I was in two mind myself. I've ditched it now. As per chat on Gophers Slack, I have a PR with Fargo to allow configurability of their logger by consumers.